README.md

```diff
@@ -284,187 +379,156 @@ The OPTIONAL `fct` field contains arbitrary facts and proofs of knowledge. The e
+For more on this representation, please refer to [§7.1](#71-canonical-json-collection).
 
-This is useful in several cases, for example:
+# 4. Reserved Resources
 
-1. When delegating all resources, like in a [`my` scheme](#4.2-all-owned-resources)
-2. To grant the maximum ability when the current ability semantics may be extended later
+The following resources are REQUIRED to be implemented.
 
-## 4.2 All Owned Resources
+## 4.1 `ucan`
 
-### 4.2.1. `my` and `as` Schemes
-
-The `my` URI scheme represents ownership over a resource — typically by parenthood — at decision-time (i.e. the validator's "now"). Resources that are created after the UCAN was created MUST be included. This higher-order scheme describes delegating some or all ambient authority to another DID.
-
-The use case of "pairing" two DIDs by delegating all current and future resources is not uncommon when a user would like to use multiple devices as "root" but does not have access to all of them directly at all times. An everyday use case for this is a user signing into multiple devices, using them both with full rights.
-
-The format for this scheme is as follows:
+The `ucan` URI subscheme defines addressing for UCANs.
 
 ``` abnf
-ownershipscheme = "my:" kind
-kind = "*" / <scheme> 
+ucan = "ucan:" ucan-selector
+ucan-selector = "*" / cid
 ```
 
-The wildcard `my:*` resource MUST be taken to mean "everything" (all resources of all types) that are owned by the current DID.
+`ucan:*` represents all of the UCANs in the current proofs array. If selecting a particular proof (i.e. not the wildcard), then the [CID](#65-content-identifiers) MUST be used. In the case of selecting a particular proof, the validator MUST check that the delegated content address is listed in the proofs (`prf`) field.
 
-A "sub-scheme" MAY be used to delegate some of that scheme controlled by parenthood. For example, `my:dns` delegates access to all DNS records. `my:mailto` selects all owned email addresses controlled by this user.
+## 4.2 `own`
 
-Redelegating these to further DIDs in a chain MUST use the `as` URI and address the specific parent DID that owns that resource, followed by the resource kind selector. For instance: `as:did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp:*` selects all resources originating from the specified DID, and `as:did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp:mailto` selects email addresses from the DID. 
+The `own` URI subscheme defines how to address all resources of a particular URI scheme owned by a particular [DID subject](https://www.w3.org/TR/did-core/#dfn-did-subjects). The DID subject MUST NOT contain any other characters, such as query parameters or fragments. The `scheme-selector` MUST either be the wildcard (`*`) selector or a URI scheme (such as `dns`, `mailto`, and `telnet`). 
 
 ``` abnf
-delegatescheme = "as:" did ":" kind
-kind = "*" / <scheme>
+own = "own://" <did-subject> "/" scheme-scope
+scheme-selector = "*" / <scheme>
```

Comment:
@expede If I recall correctly you said motivation for the `own://` protocol was due to the fact that `did:key:` can't have paths in them ? However I was unable to find that restriction & was wondering if maybe that is invalid assumption ?

I was also thinking that:

1. Query params could probably used to restrict what's delegated from one agent to the other.
2. `own:` could be subsumed by `ucan:did:key:zKey` instead so we avoid reserving more than one protocol scheme.

Reply:
There's two things:

1. UCAN isn't restricted to `did:key`. As you mention, the DID spec doesn't restrict the path segment, so any DID method can define path segment, breaking the proposed solution
2. Sure, we could use the `ucan:` scheme instead. Any thoughts pro/con on using a subscheme?

Maybe I can borrow you in person while we're in person the next few days. I'll ping you on Discord.